extensions/stackdriver/log/logger.cc

```diff
@@ -50,13 +50,17 @@ using google::protobuf::util::TimeUtil;
 
 // Name of the HTTP server access log.
 constexpr char kServerAccessLogName[] = "server-accesslog-stackdriver";
+constexpr char kAuditAccessLogName[] = "server-accesslog-stackdriver-audit";
```

Comment:
is this naming convention important?  does it need to match the historic name for the other access logs, or should we consider calling this `server-audit-log` (or something else less verbose)?

Reply:
I don't think that's important. I just went the simplest route. I'll rename it.